crates/holochain/Cargo.toml

```diff
@@ -190,7 +189,7 @@ workspace = true
 [features]
 default = ["sqlite-encrypted", "tx5", "metrics_influxive", "wasmer_sys"]
 
-tx5 = ["tx5-go-pion-turn"]
+tx5 = []
```

Comment:
Should this still be a feature flag? It's now the only option

Reply:
I'm used to leaving these in for backward compatibility... but we're breaking that anyways, so totally good time to delete it, will do :+1: